tests/storybook-config.test.js

```diff
@@ -12,6 +12,31 @@ const appDir = path.dirname(
 );
 const storybookDistDir = path.resolve(appDir, 'dist-storybook');
 
+const getStorybookElement = async (url, elementSelector) => {
+  const page = await browser.newPage();
+  await page.goto(url, { waitUntil: 'networkidle2' });
+
+  const firstStoryButton = await page.waitForSelector(
+    '#storybook-explorer-menu button',
+  );
+
+  // Ensure default story is activated
+  firstStoryButton.click();
+
+  const iframeElement = await page.waitForSelector('#storybook-preview-iframe');
+
+  const storybookFrame = await iframeElement.contentFrame();
```

Comment:
Oh yeah, this brings back [memories](https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/)

![ptsd dog](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.imgflip.com%2F4ap24b.jpg&f=1&nofb=1&ipt=f3c9a1d916200bc41d590ec71bcde31521422f791f2b5fe8c2d992bf6b91fa50&ipo=images)